src/App/Main/MainContent/NewsListView/CategoryNav.js

```diff
@@ -0,0 +1,132 @@
+/*
+기사 컨텐츠 네비게이션 컴포넌트
+*/
+const FIRST_CATEGORY = 0;
+const LAST_CATEGORY = 6;
+
+const FIRST_PAGE = 1;
+
+const categoryLengthArr = [4, 12, 3, 5, 6, 4, 7, 10];
+
+export default function CategoryNav($target, props) {
+  const initCategoryState = {
+    currentPage: FIRST_PAGE,
+    lastPage: categoryLengthArr[FIRST_CATEGORY],
+    category: FIRST_CATEGORY,
+  };
+
+  const nextCategoryState = {
+    currentPage: 1,
+    lastPage: categoryLengthArr[props.category + 1],
+    category: props.category + 1,
+  };
+
+  const nextPageState = {
+    currentPage: props.currentPage + 1,
+    lastPage: props.lastPage,
+    category: props.category,
+  };
+
+  this.startProgress = (progressBar) => {
+    const duration = 20000; // 20초
+    const startWidth = 0;
+    const endWidth = 100;
+    const startTime = performance.now();
+
+    function changeWidth(timestamp) {
+      const elapsed = timestamp - startTime;
+      const width = Math.min(
+        (elapsed / duration) * (endWidth - startWidth) + startWidth,
+        100
+      );
+
+      progressBar.style.width = width + "%";
```

Comment:
변수와 string을 함께 사용할 경우에 리터털템플릿을 사용해보면 좋을것 같아요
```suggestion
      progressBar.style.width = `${width}%`;
```